### PR TITLE
add docs for usdc bridge

### PR DESCRIPTION
**Pull request by siljapetasch**

### CODE REVIEW

**Comment by emilianobonassi** [comment]
op https://github.com/defi-wonderland/opUSDC
arb https://github.com/OffchainLabs/token-bridge-contracts/tree/main/scripts/usdc-bridge-deployment

op requires UI/partners info about new contracts
arb natively integrated

**Comment by emilianobonassi** [comment]
op ok custom documentation and refer to opusdc for more info
for arb no need, just link https://docs.arbitrum.io/build-decentralized-apps/token-bridging/token-bridge-erc20

**Comment by siljapetasch** [comment]
content is done, will double-check for typos in the morning

File: pages/usdc_bridge/overview.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+2. After deploying your new USDC bridge, make sure to update all relevant contracts and user interfaces to prevent users from interacting with an old bridge.
+
+## Bridged USDC Details
+
+The bridged USDC Token will have the following properties:
+
+- **Name:** `Bridged USDC (YOUR_ROLLUP_NAME)`
+- **Symbol:** `USDC.e`
+- **Decimals:** `6`
+- **Bridged USDC Token Address:** `TBD`
```
Comment (tadad) [inline pages/usdc_bridge/overview.mdx:38]: this they can find in the dashboard, it will vary from chain to chain

**Review by tadad — CHANGES_REQUESTED** [review]

**Review by tadad — APPROVED** [review]

**Comment by emilianobonassi** [comment]
merging for now to have link available for comms

**Pull request merged**